README.md

```diff
@@ -8,36 +8,16 @@ All Saleor services started from a single repository
 
 ## How to run it?
 
-### Mac OS
 1. Clone the repository `git clone git@github.com:mirumee/saleor-platform.git`
-2. Add the cloned `saleor-platform` directory into docker shared directories (Preferences -> Resources -> File sharing).
+2. To enable live code reload:
```

Comment:
Perhaps worth noting that on macOS and Windows it will break if you don't share it
![image](https://user-images.githubusercontent.com/12252472/75015959-261ee380-548a-11ea-8609-b0f6f4b2d550.png)